augur/application/schema/augur_full.sql

```diff
@@ -3744,7 +3744,7 @@ COPY augur_data.chaoss_metric_status (cms_id, cm_group, cm_source, cm_type, cm_b
 43	experimental	githubapi	metric	implemented	unimplemented	f	/api/unstable/<owner>/<repo>/githubapi/pull_requests_merged	\N	Pull Requests Merged	experimental	"pull_requests_merged"	Insight Worker	0.0.1	githubapi	2019-06-20 22:50:24	\N
 44	experimental	githubapi	metric	implemented	unimplemented	f	/api/unstable/<owner>/<repo>/githubapi/pull_requests_open	\N	Pull Requests Open	experimental	"pull_requests_open"	Insight Worker	0.0.1	githubapi	2019-06-20 22:50:25	\N
 45	experimental	githubapi	metric	implemented	unimplemented	t	/api/unstable/<owner>/<repo>/githubapi/repository_size	\N	Repository Size	experimental	"repository-size"	Insight Worker	0.0.1	githubapi	2019-06-20 22:50:27	\N
-46	experimental	githubapi	metric	implemented	implemented	t	/api/unstable/<owner>/<repo>/bus_factor	\N	Bus Factor	experimental	"bus-factor"	Insight Worker	0.0.1	githubapi	2019-06-20 22:50:28	\N
+46	experimental	githubapi	metric	implemented	implemented	t	/api/unstable/<owner>/<repo>/contributor_absence_factor	\N	Contributor Absence Factor	experimental	"contributor-absence-factor"	Insight Worker	0.0.1	githubapi	2019-06-20 22:50:28	\N
```

Comment:
@Preshh0 : Unfortunately making this change will be a bit more work. I know for sure there are files in : 
augur/api and docs/ that will need to be addressed.

Reply:
Alrighty then. Do I close it entirely? Or leave it open and wait until whenever there is time to work on it? @sgoggins